index.html

```diff
@@ -5,61 +5,59 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link href="https://fonts.googleapis.com/css?family=Dosis" rel="stylesheet">
     <link rel="stylesheet" href="vendor/styles/reset.css">
+    <!-- owl carousel links -->
+    <link rel="stylesheet" href="owl-carousel/assets/owl.carousel.min.css">
+    <link rel="stylesheet" href="owl-carousel/assets/owl.theme.default.css">
     <!-- <link rel="stylesheet" href="css/style.css"> -->
     <link rel="stylesheet" href="css/base.css">
-    <link rel="stylesheet" href="css/modules.css">
     <link rel="stylesheet" href="css/icons.css">
     <link rel="stylesheet" href="css/layout.css">
+    <link rel="stylesheet" href="css/modules.css">
+    <!-- jQ 3.1.1 -->
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
     <title></title>
   </head>
+
   <body>
     <!-- all elements getting a divs class assignment for formatting -->
 
     <!-- header below -->
     <nav class="nav-bar divs">
       <!-- Insert navigation items here -->
+      <a href="#table-of-contents">Projects</a>
+      <a href="#about-me">About</a>
       &nbsp;
     </nav>
 
     <header class="header divs">
-      &nbsp;
       <!-- Insert Logo and Logo text here -->
+      IMALOGOHURR
+      <nav class="nav-burger">&#9776;</nav>
     </header>
 
-    <!-- end header -->
-
-    <!-- TABLE OF CONTENTS FLOATED LEFT ASIDE -->
-    <aside class="table-of-contents divs">
-      <h3>Table of Contents</h3>
-      <ul>
-        <li>Project 1</li>
-        <li>Project 2</li>
-        <li>Project 3</li>
-      </ul>
-    </aside>
-
-    <!-- Main setup for the slideshow -->
-    <article id="slideshow-article" class="divs">
-      <!-- slideshow pictures -->
-      <div class="slideshow-div">
-        <ul class="slideshow">
-          <li class="my-slides"><img src="img/slide-01.jpg" alt></li>
-          <li class="my-slides"><img src="img/slide-02.jpg" alt></li>
-          <li class="my-slides"><img src="img/slide-03.jpg" alt></li>
-          <li class="my-slides"><img src="img/slide-04.jpg" alt></li>
-          <li class="my-slides"><img src="img/slide-05.jpg" alt></li>
-          <li class="my-slides"><img src="img/slide-06.jpg" alt></li>
-        </ul>
+      <!-- Owl Carousel Formatting -->
+      <div class="owl-carousel owl-theme divs" id="slideshow-article">
+          <div class="item"><img src="img/slide-01.jpg" alt></div>
+          <div class="item"><img src="img/slide-02.jpg" alt></div>
+          <div class="item"><img src="img/slide-03.jpg" alt></div>
+          <div class="item"><img src="img/slide-04.jpg" alt></div>
+          <div class="item"><img src="img/slide-05.jpg" alt></div>
+          <div class="item"><img src="img/slide-06.jpg" alt></div>
       </div>
+      <!-- TABLE OF CONTENTS FLOATED LEFT ASIDE -->
+      <aside class="table-of-contents divs">
+        <a name="table-of-contents">Contents</a>
```

Comment:
If you want this to display all of your projects, I would suggest just making it a section instead of an aside for now!